src/renderer/hooks/useDeeplinking.js

```diff
@@ -47,9 +46,12 @@ function useDeepLinkHandler() {
 
   const handler = useCallback(
     (event: any, deeplink: string) => {
-      const [, path] = deeplink.split("ledgerlive://");
+      const { pathname, searchParams } = new URL(deeplink);
+      const query = Object.fromEntries(searchParams);
 
-      const { url, query } = querystring.parseUrl(path);
```

Comment:
there is one thing i didn't spot: the logic no longer verify the domain is ledgerlive. I think it's necessary when we will start supporting more than just ledgerlive://

not blocking this merge, but we will need to be aware for future